knowledge-base/binaryformatter-exception-after-upgrade-to-net5.md

```diff
@@ -0,0 +1,48 @@
+---
+title: BinaryFormatter serialization and deserialization are disabled within this application is thrown in .NET5
+description: BinaryFormatter serialization and deserialization are disabled within this application after upgrade to .NET5
+type: troubleshooting
+page_title: BinaryFormatter serialization and deserialization are disabled within this application is thrown in .NET5
+slug: binaryformatter-exception-after-upgrade-to-net5
+position: 
+tags: .NET 5.0
+ticketid: 1494918
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+The following error: **Error registering the viewer with the service.
+An error has occurred.
+BinaryFormatter serialization and deserialization are disabled within this application** might be thrown after upgrade to .NET 5 in .NET Core or Blazor Projects.
```

Comment:
maybe it's better to say "after upgrade to .NET 5 **from** .NET Core or Blazor Projects."